setup.py

```diff
@@ -35,4 +34,7 @@
         "Topic :: Utilities",
     ],
     test_suite="tests",
+    entry_points={
```

Comment:
I find a locally runnable python script convenient for testing before installation.

The rename to cli.py is fine, but could you add back the lines removed, so that part works the same way as before?

Reply:
Sure can

Reply:
oh, you mean in ./py2many.py ; let me see if something similar will suffice.

Reply:
you can always do `PYTHONPATH={PWD} python3 ./py2many.py` , but I have added some glue so `PYTHONPATH` doesnt need to be set.

Reply:
ping @adsharma , how does this look now?

Reply:
Looks great. Thank you!